Library/Formula/openvx.rb

```diff
@@ -0,0 +1,35 @@
+class Openvx < Formula
+  homepage "https://www.khronos.org/openvx/"
+  url "https://www.khronos.org/registry/vx/sample/openvx_sample_20141217.tar.gz"
+  sha1 "144403e3b3e7e7f6f3a75371bfd3ca023466bbfa"
+  version "1.0"
+
+  depends_on "cmake" => :build
+
+  patch do
+    url "https://raw.githubusercontent.com/invx/homebrew-openvx/master/openvx_sample_macos.diff"
+    sha1 "3e5db79d4218c9cdf46e993f0f076ac4ee06495c"
+  end
```

Comment:
This will need to be submitted (and probably accepted) upstream before we can accept this formula.